dss/events/handlers/index.py

```diff
@@ -218,9 +221,23 @@ def notify(subscription_id: str, subscription: dict, bundle_id: str, logger):
             "bundle_version": bundle_version
         }
     }
-    # TODO (mbaumann) Ensure webhooks are only delivered over verified HTTPS (unless maybe when running a test)
     callback_url = subscription['callback_url']
-    response = requests.post(callback_url, json=payload)
+
+    # FIXME wrap all errors in this block with an exception handler
+    assert urlparse(callback_url).scheme in {"http", "https"}, "Unexpected scheme for callback URL"
```

Comment:
suggestion:

```
if os.environ["DSS_DEPLOYMENT_STAGE"] == "prod":
    allowed_schemes = {'https'}
else:
    allowed_schemes = {'https', 'http'}

assert urlparse(callback_url).scheme in allowed_schemes, "Unexpected scheme for callback URL"
```

Reply:
Also these assertions are untested.

Reply:
Thanks, I used your suggestion and added unit tests for `notify()`.